ddtrace/internal/forksafe.py

```diff
@@ -32,7 +32,11 @@ def ddtrace_after_in_child():
 
 def register(after_in_child):
     # type: (typing.Callable[[], None]) -> typing.Callable[[], None]
-    """Register a function to be called after fork in the child process."""
+    """Register a function to be called after fork in the child process.
+
+    Note that ``after_in_child`` will be called in all child processes across
```

Comment:
Do we really want this behaviour? Could this potentially lead to unwanted hooks being called on fork?

Reply:
Yeah it is rather subtle. Perhaps worth proposing as a separate change?